src/webots/nodes/utils/WbDictionary.cpp

```diff
@@ -87,20 +87,22 @@ bool WbDictionary::updateDef(WbBaseNode *&node, WbSFNode *sfNode, WbMFNode *mfNo
   } else if (useCase) {
     if (isAValidUseableNode) {
       const WbNode::NodeUse nodeUse = node->nodeUse();
-      bool match = false;
+      bool match;
```

Comment:
If defNodes.size() == 0 match will be undefined and tested at line 109...
```suggestion
      bool match = false;
```

Reply:
Reintroducing this definition is throwing error.
I will check for other solutions to avoid suppressing the error.